Multisig/UI/App/UIKit/UI Library/UIWindow+Extension.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIWindow+Extension.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 04.12.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+extension UIWindow {
+
+    // https://stackoverflow.com/questions/6131205/how-to-find-topmost-view-controller-on-ios
+    static func topMostController() -> UIViewController? {
```

Comment:
just asking: did you test this with various app states?
- switch safes opened
- detail screen opened
- safari vc opened
- safe info overlay opened
- import owner screen opened

Reply:
I have not tested it properly. We just managed to fix pushes, so I will continue on Monday with it

Reply:
I have checked all mentioned use cases.